_bibliography/peer-pubs.bib

```diff
@@ -101,4 +101,20 @@ @article{10944238
   doi      = {10.1109/MCSE.2025.3555023},
   issn     = {1558-366X},
   month    = {April}
-}
\ No newline at end of file
+}
+
+@article{yehudi_etal_2025_research_software_lifecycles,
+  title = {Research {{Software Lifecycles}} and {{Stages}}},
```

Comment:
I'm not sure extra brackets are needed here - I would probably remove them so that bibtex processing can handle cases with whatever style is appropriate for the venue

Reply:
Done in https://github.com/ser-rse-bridge/ser-rse-bridge.github.io/pull/7/commits/594535bf1dee0924afa26f35f72256a8ce16e697